pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -1519,6 +1519,16 @@ func (lbaas *LbaasV2) EnsureLoadBalancerDeleted(ctx context.Context, clusterName
 	return nil
 }
 
+// GetLoadBalancerName is an implementation of LoadBalancer.GetLoadBalancerName.
+func (lbaas *LbaasV2) GetLoadBalancerName(ctx context.Context, clusterName string, service *v1.Service) (string, error) {
+	loadBalancerName, err := lbaas.GetLoadBalancerName(ctx, clusterName, service)
```

Comment:
This can just be 
```
func (lbaas *LbaasV2) GetLoadBalancerName(ctx context.Context, clusterName string, service *v1.Service) (string, error) {
     return cloudprovider.DefaultGetLoadBalancerName()
}
```

Reply:
Ok I'll update instances of this.

Reply:
@andrewsykim should it be `return cloudprovider.DefaultGetLoadBalancerName(service), nil` ?

Reply:
I don't think `GetLoadBalancerName` needs to return error (weird saying that) which I see you already did :)